.travis.yml

```diff
@@ -61,7 +61,7 @@ install:
   - rm -rf "$HOME/.m2/repository/org/apache/beam"
 
 script:
-  - travis_retry mvn --batch-mode --update-snapshots --no-snapshot-updates $MAVEN_OVERRIDE install && travis_retry bash -ex .travis/test_wordcount.sh
+  - travis_retry mvn --batch-mode --update-snapshots $MAVEN_OVERRIDE install && travis_retry bash -ex .travis/test_wordcount.sh
```

Comment:
I think this may cause problems. I'll try to look into the references for the behavior here, but I think it will cause us to go to Maven Central for snapshots that we don't want to. This is to get the latest gearpump snapshot, right?